flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -128,10 +128,11 @@
 		// workaround for annoying InterruptedException logging:
 		// https://issues.apache.org/jira/browse/YARN-1022
 		"java.lang.InterruptedException",
-		// very specific on purpose
+		// very specific on purpose; whitelist meaningless exceptions that occur during akka shutdown:
 		"Remote connection to [null] failed with java.net.ConnectException: Connection refused",
 		"Remote connection to [null] failed with java.nio.channels.NotYetConnectedException",
 		"java.io.IOException: Connection reset by peer",
+		"Caused by: [java.net.ConnectException: Connection refused:",
```

Comment:
I'm wondering whether we should rework this to work on `Patterns` instead; then we could also include `akka.remote.ReliableDeliverySupervisor`, or in fact the entire message.